### PR TITLE
Fix for #303

### LINKED ISSUE #303: Google storage provider with existing bucket
**Is this a request for help?**:
YES

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

possibly a BUG

**Version of Helm and Kubernetes**:
1.11.7

**Which chart**:
0.4.4 artifactory-ha

**What happened**:

We are trying to activate google storage binary provider. As part of the setup we provision the bucket via terraform and then deploy artifactory ha helm chart.  I have set the <bucketExists> field to true in the artifactory-binarystore.yaml as suggested [here](https://www.jfrog.com/confluence/display/RTF/Configuring+the+Filestore#ConfiguringtheFilestore-GoogleStorageBinaryProvider). However when I deploy the pod in the primary statefulset does not stabilise. The following messages are in the logs

[art-init] [INFO ] (o.a.a.f.t.j.g.GoogleJets3tBinaryProvider:94) - Bucket exists = false
[ERROR] (o.a.a.f.t.j.g.GoogleJets3tBinaryProvider:274) - Failed to create bucket bucket-name on GCS --> BucketNameUnavailable:The requested bucket name is not available. The bucket namespace is shared by all users of the system. Please select a different name and try again.

**What you expected to happen**:

Should respect the bucketExists flag and not try to recreate it

**How to reproduce it** (as minimally and precisely as possible):

Enable google-storage provider and pass it existing bucket name and interoperability API credentials and try to deploy it

### PR DESCRIPTION
**Pull request by jainishshah17**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #303 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by eldada** [comment]
@jainishshah17 - please rebase.

**Comment by eldada** [comment]
@danielezer - can you have a look?

**Comment by danielezer** [comment]
Yes @eldada already did. Just one comment - shouldn't the bucketExists in values.yaml be a Boolean instead of a string? Why do we need the quotes?

File: stable/artifactory-ha/values.yaml [hunk 1]
```diff
@@ -375,6 +375,7 @@ artifactory:
       identity:
       credential:
       path: "artifactory-ha/filestore"
+      bucketExists: "false"
```
[the remark this answers is not in the document]
Comment (jainishshah17) [reply stable/artifactory-ha/values.yaml]: @danielezer You are right. just fixed it.

**Pull request merged**